website/src/consts/types.ts

```diff
@@ -1,3 +1,95 @@
 import outputJSONData from '../output/output.json'
 
-export type OutputJSONType = typeof outputJSONData
+// TODO: what if there are more islands - dynamic typing
+// eslint-disable-next-line no-shadow
+export enum TeamName {
+    'CommonPool',
+    'Team1',
+    'Team2',
+    'Team3',
+    'Team4',
+    'Team5',
+    'Team6',
+}
+
+export type Transaction = {
+    from: TeamName
+    to: TeamName
+    amount: number
+}
+
+// TODO: Extract summary metric for bubble size from transactions[] and islandGifts[]
+// TODO: might be cool to have max and min resources of each entity as a summary metric in the tooltip
+export type Node = {
+    id: number
```

Comment:
i saw `.id === 0` implying CommonPool above. should explain what Node this is for, and what the Link below is for. This is a shared util folder so it would be better to name it specific for its usecase to prevent confusion.

Reply:
Moved out of util file.